src/main/java/teammates/ui/controller/StudentCommentsPageAction.java

```diff
@@ -70,15 +70,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         }
         
         data = new StudentCommentsPageData(account);
-        data.courseId = courseId;
-        data.courseName = courseName;
-        data.coursePaginationList = coursePaginationList;
-        data.comments = comments;
-        data.roster = roster;
-        data.previousPageLink = previousPageLink;
-        data.nextPageLink = nextPageLink;
-        data.studentEmail = studentEmail;
-        data.feedbackResultBundles = feedbackResultBundles;
+        data.init(courseId, courseName, coursePaginationList, comments, roster,
+                  previousPageLink, nextPageLink, studentEmail, feedbackResultBundles);
```

Comment:
While it's slightly better than that previous one, several of these can still be interpreted in `*Data` using the other variables.